### PR TITLE
[WIP]Added export to .zip when not logged in (#981)

### PR DESCRIPTION
**Pull request by simon66**
Still needs approval and still needs to change the icon :)
As title says, WIP

PS: Also, I'm thinking of changing the export id from "export-project-zip" to "filetree-pane-nav-export-project-zip". If I do that, I'll have to modify bramble-ui-bridge.js too

### CODE REVIEW

File: locales/en-GB/messages.properties [hunk 1]
```diff
@@ -86,16 +86,16 @@ fileSavingIndicator=Saving...
 fileSavingFailedIndicator=Saving failed, retrying...
 help=Help
 deleteProjectLink=Delete Project
-exportProjectLink=Export (.zip)
+exportProjectLink=Download files (.zip)
```
Comment (Pomax) [inline locales/en-GB/messages.properties]: I think we generate this file from pontoon, with only the en-US file being "Stable". Can you confirm this @gideonthomas?

Comment (gideonthomas) [inline locales/en-GB/messages.properties]: Correct! @simon66 this change should be made in the properties file in `en-US` not `en-GB`

**Comment by gideonthomas** [comment]
Feel free to go ahead and change the `id` of the element as well

**Comment by gideonthomas** [comment]
@flukeout do you have the icon that we can use for this?

**Comment by flukeout** [comment]
@simon66 Here's the icon https://drive.google.com/open?id=0B_CPRh4f3yyqQjBRaG5ud3VDUkE

It won't show up in the preview as it's white, but that should work. Let me know if you had any trouble. I tested it out locally and added a ``background-size: 12px`` rule to the div that has the current background image (the plus) and that worked pretty well.

**Comment by simon66** [comment]
screen shot 2017-02-06 at 8 35 23 pm

Done :)

**Comment by flukeout** [comment]
For some reason the SVG is black in Firefox...

image

**Comment by flukeout** [comment]
@simon66 - I've fixed up the image and tested that it displays properly in Firefox, I have no idea what was going on with it there.

New image - https://drive.google.com/open?id=0B_CPRh4f3yyqWWxRaUdWMzVtT0U

Additionally, please update the background size on the div that contains the icon to:

```
background-size: 13px 15px;
```

Thanks for all your patience! I think we're done after this. I'll fix up the British english translations once we land this.

**Comment by simon66** [comment]
@flukeout - Thanks for the new icon :) I hope I didn't miss anything (if I did, let me know 👍 ).

**Comment by gideonthomas** [comment]
@flukeout can we land this now?
Also, if you're fixing translations, could you do it via Pontoon?

**Comment by flukeout** [comment]
Merged - thank you @simon66 !!

**Pull request merged**